### ISSUE TITLE
Scope number 0 output no rows

### ISSUE THREAD

**Issue by Wapaca** [wharfkit/contract#53]
Minor issue, if scope is 0 it must be a string otherwise it will output nothing. This bug doesn't trigger for number scopes above 0.

Test code: 
```
import {APIClient} from '@wharfkit/antelope'
import {ContractKit} from '@wharfkit/contract'

const client = new APIClient({url: 'https://wax.greymass.com'})

const contractKit = new ContractKit({client})

const contract = await contractKit.load('alcordexmain')

const rowsBuyIntScope = await contract.table('buyorder').query({ scope: 0 }).all()
const rowsSellIntScope = await contract.table('sellorder').query({ scope: 0 }).all()

const rowsBuyTxtScope = await contract.table('buyorder').query({ scope: '0' }).all()
const rowsSellTxtScope = await contract.table('sellorder').query({ scope: '0' }).all()

// Length is 0
console.log(rowsBuyIntScope.length, rowsSellIntScope.length)

// Length is > 0
console.log(rowsBuyTxtScope.length, rowsSellTxtScope.length)
```

**Comment by aaroncox** [comment]
Good catch. Typescript would have noticed and refused to compile in this instance, but Javascript let it pass by and wasn't setting the scope on the request because the `0` value is null-ish.

We'll have a fix in the next release 👍

**Comment by aaroncox** [comment]
Released in 0.4.2!
